Documentation/network/servicemesh/gateway-api/host-network-mode.rst

```diff
@@ -0,0 +1,70 @@
+Host network mode
+#################
+.. note::
+    Supported since Cilium 1.16+
+
+This mode allows you to expose the Cilium GatewayAPI Gateway
+directly on the host network.
+This is useful in some edge cases where one doesn't want, or can't, use
+LoadBalancer type services (dev environments, cluster-external
+loadbalancer, ...).
+
+The host network mode can be enabled via Helm flag
+``gatewayAPI.hostNetwork.enabled=true``.
+Once enabled, the host network ports can be specified on the ``Gateway``
+resources via ``spec.listeners.port``. The port must be unique per ``Gateway``
+resource and higher than ``1023``.
+
+.. warning::
+    * Enabling the Cilium GatewayAPI host network mode automatically disables the LoadBalancer/NodePort type service mode. They are mutually exclusive.
+    * The listener is exposed on all interfaces (``0.0.0.0`` for IPv4 and/or ``::`` for IPv6).
+    * Be aware that misconfiguration might result in port clashes. Configure unique ports that are still available.
+    * It's mandatory to use ports higher than ``1023`` as the Cilium L7 proxy process `does not have any Linux capabilities by default`_ (so no ``CAP_NET_BIND_SERVICE``) and is therefore not allowed to listen on privileged ports. If you choose a port equal to or lower than ``1023``, ensure that ``envoy.securityContext.capabilities.keepCapNetBindService=true`` is configured and to add the capability ``NET_BIND_SERVICE`` to the respective :ref:`Cilium Envoy container via Helm values<envoy>`:
+
+        * Standalone DaemonSet mode: ``envoy.securityContext.capabilities.envoy``
+        * Embedded mode: ``securityContext.capabilities.ciliumAgent``
+
+.. _does not have any Linux capabilities by default: https://github.com/cilium/proxy/pull/315
```

Comment:
is this still required? it seems as this doesn't get rendered at all https://deploy-preview-31839--docs-cilium-io.netlify.app/network/servicemesh/gateway-api/gateway-api#host-network-mode

Reply:
Not really. Let's remove it.